tests/dat/actions/largeFile.js

```diff
@@ -0,0 +1,11 @@
+function main (params) {
+    var name = params.payload || "User";
+    var list = "L69lhnfnD1rjFoesRquHOIXHAMR3cYp8pVxfag38WNHn6ijlyEn7r5VsNlM6kvZQHje6q6nTV5Wts3wwG6NqQGVllPm5bH6cFnburN2UwRIZrVVK9lPXTlL0YmZbpKQaTilpiXhnQH2RwRGEtzK2Puos\n" +
```

Comment:
Could do something like this var list = "a".repeat(1000)` or `var list = Array(1000).join("a")`.